app/controllers/api/v1/registrations_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Api::V1::RegistrationsController < Api::V1::BaseController
+  before_action :set_user, only: [:edit, :update,:destroy]
```

Comment:
:edit  -> This value is of which method ?, Should be removed if method is not there